docs/UserGuide.md

```diff
@@ -75,10 +75,24 @@ use via a CLI while still having the benefits of a GUI.
     - If a parameter is expected only once in the command, but you specified it multiple times,
       an error message will be shown.
       
-    - If an extraneous parameter is specified, an error message will be shown.
+    - If an extraneous parameter is specified, an error message will be shown. (Does not apply for tags or contacts)
+    
+    - Parameter or argument values cannot contain the character `/`. An error message will be shown if `/` is present in the argument value. For example, `task add 1/2` is invalid, and can be replaced with valid alternatives such as `task add 1 or 2` instead.
```

Comment:
`task add 1` or `task add 2`

Reply:
wait what no, this example is talking about if like people wanna have multiple choices in their task title, not about multiple tasks.

For eg) `task add Bake cake/cookies` will have to become `task add Bake cake or cookies`

Reply:
Right.